app-ios/Sources/MapFeature/MapView.swift

```diff
@@ -39,9 +36,10 @@ public struct MapView: View {
         WithViewStore(store) { viewStore in
             NavigationView {
                 VStack(alignment: .center) {
```

Comment:
`VStack` already exists, but is this necessary?
If you don't mind, can you remove that and align image center? 🙏🏼

Reply:
I fixed in 81a7045e9ffab049650f1461d1ca1cd644d50d4e